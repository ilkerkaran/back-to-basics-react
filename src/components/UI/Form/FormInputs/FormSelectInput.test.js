import React from 'react';
import renderer from 'react-test-renderer';
import Select from './FormSelectInput';


describe('<Select /> snapshot ', () => {
  const inputConfig = {
    inputType: 'select',
    inputName: 'paymentType',
    label: 'Please select a Payment Type',
    isRequired: true,
    options: [{
      id: 'creditCard',
      value: 'Credit/Debit Card'
    }, {
      id: 'cash',
      value: 'Cash'
    }],
    register: () => {},
    errors: {}
  };


  it('renders correctly', () => {
    const tree = renderer
      .create(<Select inputConfig={inputConfig}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
