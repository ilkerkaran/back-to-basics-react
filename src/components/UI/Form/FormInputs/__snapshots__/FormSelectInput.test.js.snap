// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Select /> snapshot  renders correctly 1`] = `
<div>
  <select
    className="Input"
    name="paymentType"
    onChange={[Function]}
  >
    <option
      value=""
    >
      Please select a Payment Type
    </option>
    <option
      value="creditCard"
    >
      Credit/Debit Card
    </option>
    <option
      value="cash"
    >
      Cash
    </option>
  </select>
  <div
    style={
      Object {
        "color": "darkred",
      }
    }
  />
</div>
`;
